src/DependencyInjection/Factory/RoutingAdminFactory.php

```diff
@@ -35,16 +35,22 @@ public function getKey()
     public function addConfiguration(NodeBuilder $builder)
     {
         $builder->scalarNode('basepath')->defaultNull()->end();
+        $builder->scalarNode('content_basepath')->defaultNull()->end();
     }
 
     /**
      * {@inheritdoc}
      */
     public function create(array $config, ContainerBuilder $container, XmlFileLoader $loader)
     {
+        if (!$config['enabled']) {
```

Comment:
This should be reverted. `AdminFactory#create()` is not called when the factory is disabled: https://github.com/symfony-cmf/sonata-phpcr-admin-integration-bundle/blob/65036a3c48e214c78df9460b156ad052eff1d36b/src/DependencyInjection/CmfSonataPhpcrAdminIntegrationExtension.php#L81